packages/babel-preset-mc-app/index.js

```diff
@@ -60,6 +61,9 @@ module.exports = function getBabePresetConfigForMcApp() {
           // Will use the native built-in instead of trying to polyfill
           // behavior for any plugins that require one.
           useBuiltIns: true,
+          // Allows changing the JSX runtime.
+          // https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
+          runtime: jsxRuntime,
```

Comment:
In react-scripts, they define `useBuiltIns` only when NOT using the runtime: https://github.com/facebook/create-react-app/blob/ed958938f642007645dd5ac3466db36202f8754e/packages/babel-preset-react-app/create.js#L98-L99

Reply:
Thanks. Will add!